__tests__/app.test.js

```diff
@@ -119,13 +119,72 @@ describe("GET /api/articles", () => {
         expect(articles).toBeSortedBy('created_at', { descending: true })
       })
   })
+})
 
-  /*test("500: Responds with an error message when there is a server error", () => {
+describe("GET /api/articles/:article_id/comments", () => {
+  test("200: Responds with an array of comments for the given article_id", () => {
     return request(app)
-      .get("/api/articles")
-      .expect(500)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then(({ body: { comments } }) => {
+        expect(comments).toBeInstanceOf(Array)
+        if (comments.length > 0) {
```

Comment:
There's no need for any conditional logic in here. I would instead have a length check before the forEach, then if that failed, we know there's something wrong 👍